intelmq/bots/BOTS

```diff
@@ -653,6 +103,45 @@
                 "xmpp_server": "xmpp server",
                 "xmpp_user": "xmpp username"
             }
+        },
+        "AlienVault OTX": {
```

Comment:
@sykaeh to drop the sorting you have change test_BOTS_syntax() in intelmq/tests/test_conf.py . From reading the code it seems that you would need to replace the call to the custom to_json() function with an explicit call that does not sort.